smokeflow.rb

```diff
@@ -0,0 +1,118 @@
+require 'rspec'
```

Comment:
maybe you could add smoke test cases at beginning e.g.
#Test cases:
#1. Login to app ....